WcaOnRails/lib/auxiliary_data_computation.rb

```diff
@@ -2,27 +2,31 @@
 
 module AuxiliaryDataComputation
   def self.compute_everything
-    self.compute_best_of_3_in_333bf
+    self.compute_mean_for_bo3_as_mo3_events
     self.compute_concise_results
     self.compute_rank_tables
 
     self.delete_php_cache # Note: this should go away together with the PHP code.
   end
 
-  ## Compute mean for 'best of 3' results in 333bf.
-  def self.compute_best_of_3_in_333bf
+  ## Compute mean for 'best of 3' results in 333bf/444bf/555bf.
+  def self.compute_mean_for_bo3_as_mo3_events
     # Set new DNF average where any of three solves is not completed.
-    Result.where(eventId: "333bf", formatId: "3", average: 0)
+    Result.where(eventId: ["333bf", "444bf", "555bf"], formatId: "3", average: 0)
           .where("LEAST(value1, value2, value3) < 0")
           .where.not(value1: 0, value2: 0, value3: 0)
           .update_all(average: -1)
     # Set new averages (round times > 10:00).
-    Result.where(eventId: "333bf", formatId: "3", average: 0)
+    Result.where(eventId: ["333bf", "444bf", "555bf"], formatId: "3", average: 0)
```

Comment:
nit: I think it would be nice to pull this out into a `bo3_as_mo3_events` variable, and reference that here and on line 15.